src/app/child-dev-project/children/children-list/list-relation/list-relation.component.html

```diff
@@ -0,0 +1,3 @@
+
+<app-school-block *ngIf="id === 'schoolId'" [entityId]="relation?.schoolId"></app-school-block>
```

Comment:
maybe just inline this in the list-component, to reduce the number of components a bit?